### PR TITLE
Betterbird.Betterbird version 102.6.0-bb26

### PR DESCRIPTION
**Pull request by borque**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.2 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.2.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/91768)

### CODE REVIEW

**Comment by borque** [comment]
Please see PRs #89294 and #90534 where a similar unexplained error occured and @stephengillie added a waiver for the  Validation-Installation-Error

File: manifests/b/Betterbird/Betterbird/102.6.0-bb26/Betterbird.Betterbird.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  InstallerSha256: A4199D1595FDCFCCDED64513F8228CB921B8105A0A815D0116D1A3B976C648CB
+- InstallerLocale: es-AR
+  Architecture: x64
+  InstallerUrl: https://www.betterbird.eu/downloads/WindowsInstaller/betterbird-102.6.0-bb26.es-AR.win64.installer.exe
+  InstallerSha256: 13167373406C405823624C37969BCBA5C1C23005C738B367045B6A21D2AEF025
+- InstallerLocale: pt-BR
+  Architecture: x64
+  InstallerUrl: https://www.betterbird.eu/downloads/WindowsInstaller/betterbird-102.6.0-bb26.pt-BR.win64.installer.exe
+  InstallerSha256: 0B1BFA05513A590E7347705E19957268F5994AD1BA89089A579F4592AC1C4194
+- InstallerLocale: ja-JA
```
Comment (stephengillie) [inline manifests/b/Betterbird/Betterbird/102.6.0-bb26/Betterbird.Betterbird.installer.yaml:40]:
```suggestion
- InstallerLocale: ja-ja
```

Comment (stephengillie) [reply manifests/b/Betterbird/Betterbird/102.6.0-bb26/Betterbird.Betterbird.installer.yaml:40]: Validation is erroring while validating the Japanese-Japan locale install. It seems to be mismatching caps, searching for `ja-ja` but finding `ja-JA` among others: 
```
2022-12-18 11:39:26.681 [CLI ] Installer [X64,exe,Unknown,ja-JA] not applicable: Installer locale does not match required locale: ja-JARequired locales: [ja-ja]
```

Additionally, there is no Betterbird.Betterbird.locale.ja-JA.yaml nor Betterbird.Betterbird.locale.ja-ja.yaml included in this manifest.

**Comment by stephengillie** [comment]
> Please see PRs #89294 and #90534 for previous releases of Betterbird where a similar unexplained error occured and @stephengillie added a waiver for the Validation-Installation-Error

Validation is erroring while validating the Japanese-Japan locale install. It seems to be mismatching caps, searching for `ja-ja` but finding `ja-JA` among others: 
```
2022-12-18 11:39:26.681 [CLI ] Installer [X64,exe,Unknown,ja-JA] not applicable: Installer locale does not match required locale: ja-JARequired locales: [ja-ja]
```

Additionally, there is no Betterbird.Betterbird.locale.ja-JA.yaml nor Betterbird.Betterbird.locale.ja-ja.yaml included in this manifest.

**Comment by borque** [comment]
> Validation is erroring while validating the Japanese-Japan locale install. It seems to be mismatching caps, searching for `ja-ja` but finding `ja-JA` among others:

If this simple change ja-JA -> ja-ja is the solution, it would be great. I will implement this in future updates.

> Additionally, there is no Betterbird.Betterbird.locale.ja-JA.yaml nor Betterbird.Betterbird.locale.ja-ja.yaml included in this manifest.

Unfortunately I don't have any idea of the Japanese language. And while I could partitially rely on my own knowledge and mostly on DeepL and Betterbirds project leader for the European languages the Japanese translators haven't provided a Description yet and no one of us can judge DeepLs translation...

**Review by stephengillie — APPROVED** [review]

**Pull request merged**